helm/idol-community/Chart.yaml

```
@@ -0,0 +1,27 @@
+# BEGIN COPYRIGHT NOTICE
+# Copyright 2023 Open Text.
+# 
+# The only warranties for products and services of Open Text and its affiliates and licensors
+# ("Open Text") are as may be set forth in the express warranty statements accompanying such
+# products and services. Nothing herein should be construed as constituting an additional warranty.
+# Open Text shall not be liable for technical or editorial errors or omissions contained herein.
+# The information contained herein is subject to change without notice.
+#
+# END COPYRIGHT NOTICE
+apiVersion: v2
+name: idol-community
+version: "0.1.0"
+appVersion: "23.4"
+description: Adds IDOL Community
+dependencies:
+- name: idol-library
+  version: "0.2.0"
+  repository: https://raw.githubusercontent.com/opentext-idol/idol-containers-toolkit/main/helm
+- name: idol-licenseserver
+  condition: idol-licenseserver.enabled
+  version: "0.1.0"
+  repository: https://raw.githubusercontent.com/opentext-idol/idol-containers-toolkit/main/helm
+- name: single-content
+  condition: idol-licenseserver.enabled
```

```suggestion
  condition: single-content.enabled
```
(as per values file)?